python: can't open file '/home/khushal/tempProbo/AlgoTrading/ProboAPI/youtubeTrading/utilt.py': [Errno 2] No such file or directory
python: can't open file '/home/khushal/tempProbo/AlgoTrading/ProboAPI/youtubeTrading/utitls.py': [Errno 2] No such file or directory
Error: 403, {
  "error": {
    "code": 403,
    "message": "The request cannot be completed because you have exceeded your \u003ca href=\"/youtube/v3/getting-started#quota\"\u003equota\u003c/a\u003e.",
    "errors": [
      {
        "message": "The request cannot be completed because you have exceeded your \u003ca href=\"/youtube/v3/getting-started#quota\"\u003equota\u003c/a\u003e.",
        "domain": "youtube.quota",
        "reason": "quotaExceeded"
      }
    ]
  }
}

Traceback (most recent call last):
  File "/home/khushal/tempProbo/AlgoTrading/ProboAPI/youtubeTrading/utils.py", line 49, in <module>
    # cronUpdateViews()
  File "/home/khushal/tempProbo/AlgoTrading/ProboAPI/youtubeTrading/utils.py", line 41, in cronUpdateViews
    data["history_view_rate"].append({ currTime : currViews - lastViews })
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
